lib/sejfguru_web/templates/layout/navbar.html.eex

```diff
@@ -0,0 +1,28 @@
+<%= if assigns[:current_user] do %>
+  <header class="header">
+    <nav class="navbar navbar-inverse navbar-fixed-top">
+      <div class="container-fluid">
+        <div class="navbar-header">
+          <a class="navbar-brand" href="/">Sejfguru</a>
+        </div>
+        <ul class="nav navbar-nav">
+          <li class="active"><a href="<%= asset_path(@conn, :index) %>">Assets</a></li>
+        </ul>
+        <ul class="nav navbar-nav navbar-right">
+          <li class="dropdown">
+            <a href="" class="dropdown-toggle" id="dropdownNavbar"
+                    data-toggle="dropdown" aria-haspopup="true" aria-expanded="true"
```

Comment:
I'd align `data-toggle` as `href` is.

Reply:
We need some linter for stuff like this 😞